doc/release-process.md

```diff
@@ -106,15 +108,13 @@ Notify the Zcash DevOps engineer/sysadmin that the release has been tagged. They
 
 Then, verify that nodes can connect to the testnet server, and update the guide on the wiki to ensure the correct hostname is listed in the recommended zcash.conf.
 
-## F. publish the release announcement (blog, zcash-dev, slack)
-## G. celebrate
+## F. Update the Beta Guide
```

Comment:
Can we add post-release steps of running the Gitian build and building the .deb packages?